app-utils/keyd/autobuild/overrides/usr/lib/sysusers.d/keyd.conf

```diff
@@ -0,0 +1 @@
+g keyd
```

Comment:
Seems like the user would need to configure their group property to make use of keyd? If so, please introduce a [debconf prompt template](http://www.fifi.org/doc/debconf-doc/tutorial.html) to notify the user.

You might reference `app-devel/intel-oneapi-basekit` for how this is done.

Reply:
It's used for an experimental feature. Personally I think it's a niche feature and shouldn't bother a normal user for it.